CRM/Member/Form/MembershipType.php

```diff
@@ -187,12 +187,11 @@ public function buildQuickForm() {
 
     $membershipRecords = FALSE;
     if ($this->_action & CRM_Core_Action::UPDATE) {
-      $membershipType = new CRM_Member_BAO_Membership();
-      $membershipType->membership_type_id = $this->_id;
-      if ($membershipType->find(TRUE)) {
-        $membershipRecords = TRUE;
-        $memberRel->freeze();
-      }
+        $result = civicrm_api3("Membership", "get", ["membership_type_id"=>$this->_id, "options"=>["limit"=>1]]);
+        $membershipRecords = $result["count"] > 0;
+        if ($membershipRecords) {
+            $memberRel->freeze();
```

Comment:
@JO0st 2 things this you will want to still assign $membershipRecords as it gets passed to the template at L197 and also CiviCRM style checker won't pass the array sort version need it in the long format array()